pkg/controller/registry/reconciler/reconciler.go

```diff
@@ -4,13 +4,14 @@ package reconciler
 import (
 	"strings"
 
+	v1 "k8s.io/api/core/v1"
```

Comment:
I'd like to see us standardize these package import names and have all of these be `corev1` at some point.

I'll briefly hold in case someone else wants to review but these changes sound reasonable to me.

/lgtm
/hold

Reply:
I blindly use https://github.com/AanZee/goimportssort as a file watcher in Goland, so I don't pay much attention to the imports block. It probably wouldn't be too hard to write a simple check that looks for some known api packages and checks that the import name follows that pattern.